oneoffs/prepare_bigquery.py

```diff
@@ -139,16 +142,22 @@ def extract_move_data(root_node, worker_id, completed_time, board_size):
             import pdb; pdb.set_trace()
         move_played = coords.to_flat(coords.from_sgf(move_played))
         post_Q, debug_rows = parse_comment_node(props['C'][0])
-        policy_prior = [0] * (board_size * board_size + 1)
-        policy_prior_orig = policy_prior[:]
-        mcts_visit_counts = policy_prior[:]
-        mcts_visit_counts_norm = policy_prior[:]
-        for debug_row in debug_rows:
-            move = debug_row.move
-            policy_prior[move] = debug_row.prior
-            policy_prior_orig[move] = debug_row.orig_prior
-            mcts_visit_counts[move] = debug_row.N
-            mcts_visit_counts_norm[move] = debug_row.soft_N
+
+        def get_row_data(debug_row):
+            column_names = ["prior", "orig_prior", "N", "soft_N"]
+            return [getattr(debug_row, field) for field in column_names]
+
+        if FLAGS.only_top_move:
+            assert len(debug_rows) <= 1
+            row_data = list(map(get_row_data, debug_rows))
+        else:
+            row_data = [[0,0,0,0],] * (board_size * board_size + 1)
```

Comment:
i'm confused about this trailing comma.
Also, leave a comment that yes, this list is shared between all zero'd row data entries, but since they don't get mutated, it doesn't matter?